assets/queries/terraform/alicloud/nas_file_system_without_kms/query.rego

```diff
@@ -0,0 +1,15 @@
+package Cx
+
+CxPolicy[result] {
+	resource := input.document[i].resource.alicloud_nas_file_system[name]
+	resource.encrypt_type == "2"
+	not resource.kms_key_id
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("alicloud_nas_file_system[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("alicloud_nas_file_system[%s].kms_key_id' is defined'", [name]),
+		"keyActualValue": sprintf("alicloud_nas_file_system[%s].kms_key_id' is undefined", [name]),
+	}
+}
```

Comment:
I think we should rethink this query approach.

When `encrypt_type` is set to `2`, `kms_key_id `is required. So, the present policy can be considered as unnecessary.

I suggest two policies. One that verifies if `encrypt_type` is not set, which means that the resource is not encrypted. And another policy that verifies if `encrypt_type` is not set to `2`.

Please:
- Rethink the approach
- Include `searchLine`
- Improve query description to point that `encrypt_type` should be set to `2`